components/util/BackButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "../../styles/class.module.css";
```

Comment:
this should not be styles from the `class.module.css`. This component should have its own styles (or no styles). The place where you use this component should have the styling for this